libvast/vast/system/index.hpp

```diff
@@ -83,7 +83,7 @@ struct index_state {
 
   /// Stores evaluation metadata for pending partitions.
   using pending_query_map
-    = caf::detail::unordered_flat_map<uuid, evaluation_map>;
+    = caf::detail::unordered_flat_map<uuid, evaluation_triples>;
```

Comment:
Can we change the type of `unordered_flat_map` here as well? Or is there a clear benefit for using it?

Reply:
Will do.